iphone/Maps/Classes/CarPlay/CarPlayService.swift

```diff
@@ -654,7 +654,7 @@ extension CarPlayService {
   }
 
   func createEstimates(routeInfo: RouteInfo) -> CPTravelEstimates? {
-    if let distance = Double(routeInfo.targetDistance) {
+    if let distance = NumberFormatter().number(from: routeInfo.targetDistance)?.doubleValue {
```

Comment:
It's ok as a quick workaround. But why the distance is passed as a string, and then converted to a double? Is it possible to store raw double in the RouteInfo?

Reply:
Yes that's much better. I'll post my changes soon.

Reply:
I did some quick tests and it worked in the simulator. I'm just confused why they used a string instead of the double. This needs maybe more testing...

Reply:
@fabwu is it hard to refactor this code a bit in a separate PR, to avoid this unnecessary and risky conversion?